var/spack/repos/builtin/packages/py-psyclone/package.py

```diff
@@ -67,6 +68,9 @@ class PyPsyclone(PythonPackage):
     depends_on("py-pytest-xdist", type="test")
```

Comment:
There appear to be more test dependencies here than needed by the latest version per `setup.py`:

```
        extras_require={
            'doc': ["sphinx", "sphinxcontrib.bibtex", "sphinx-tabs",
                    "sphinx_rtd_theme", "sphinx-autodoc-typehints", "autoapi"],
            'test': ["flake8", "pylint", "pytest-cov", "pytest-xdist"],
        },
```

Reply:
Oh yes, simplified.

Reply:
No, they're not needed. I don't know why they were there - we didn't use those packages from within pytest.